src/command_modules/azure-cli-vsts/azure/cli/command_modules/vsts/custom.py

```diff
@@ -192,49 +197,73 @@ def list_releases(name, resource_group_name):
     json_request = req.json()
     return json_request
 
+def _gitroot():
+    """
+    returns the absolute path of the repository root
+    """
+    try:
+        base = check_output(['git', 'rev-parse', '--show-toplevel'])
+    except:
+        raise IOError('Current working directory is not a git repository')
+    return base.decode('utf-8').strip()
+
+def _get_filepath_in_current_git_repo(filename):
+    """
+    retrieves the full path of the first file in the git repo that matches filename
+    """
+    FILE_NAMES = 2
+    CWD = 0
+    for walk in os.walk(_gitroot()):
+        for file_name in walk[FILE_NAMES]:
+            if file_name == filename:
+                return walk[CWD] + '/' + file_name
+    return False
+
 def _ensure_docker_compose():
     """
     1. Raises an error if there is no docker_compose_file present.
     2. Raises an error if the version specified in the docker_compose_file is not
     docker_compose_version.
     3. Raises an error if docker_compose_test_file has a version other than docker_compose_version.
     """
-    docker_compose_file = 'docker-compose.yml'
-    docker_compose_test_file = 'docker-compose.test.yml'
-    docker_compose_expected_version = '2'
-    if not os.path.isfile(docker_compose_file):
-        raise CLIError('Docker compose file "{}" was not found.'.format(docker_compose_file))
+    docker_compose_file = _get_filepath_in_current_git_repo(DOCKER_COMPOSE_FILE)
+    docker_compose_test_file = _get_filepath_in_current_git_repo(DOCKER_COMPOSE_TEST_FILE)
+
+    if not docker_compose_file:
+        raise CLIError('Docker compose file "{}" was not found.'.format(DOCKER_COMPOSE_FILE))
     with open(docker_compose_file, 'r') as f:
         compose_data = yaml.load(f)
         if 'version' not in compose_data.keys():
```

Comment:
checking for version and is exactly the same for both files - consider refactoring (e.g. ```def ensure_version(file_name, expected_version```)